src/components/HeatMap/Cell/Cell.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Wrapper } from './Cell.styles';
+
+
+export default function Cell({
+  itemsNumber, row, column, activeCell, getActiveCell,
+}) {
+  const [isActive, setActive] = useState(false);
+
+  useEffect(() => {
+    if (activeCell[0] === row && activeCell[1] === column) {
+      setActive(true);
+      return;
+    }
+    setActive(false);
+  }, [activeCell, row, column]);
+
+  const handleClick = () => {
+    getActiveCell(row, column);
```

Comment:
The name `getActiveCell` is misleading. I left a comment in the parent component how it can be improved